generate_ics.py

```diff
@@ -19,14 +19,14 @@
 WORKING_DAYS = dates.get_dates()
 
 import build_event
-from update_subjects_json import update_sub_list
 
 import argparse
 import getpass
 
 parser = argparse.ArgumentParser()
 parser.add_argument("-i", "--input")
 parser.add_argument("-o", "--output")
+parser.add_argument("--session", default="2023-2024")
```

Comment:
Mention about these changes in `README.md`

Reply:
My bad again, this argument is not required anymore, removing it